src/core_plugins/kbn_vislib_vis_types/public/pie.js

```diff
@@ -100,6 +98,6 @@ export default function HistogramVisType(Private) {
       ])
     },
     hierarchicalData: true,
-    responseConverter: buildHierarchicalData,
+    responseHandler: 'vislib_slices',
```

Comment:
vislib now registers two response handlers: vislib_slices and vislib_series ....
responseConverter was removed